addon/globalPlugins/webAccess/ruleHandler/__init__.py

```diff
@@ -1278,6 +1278,8 @@ def load(self, data):
 		self.relativePath = data.pop("relativePath", None)
 		self.index = data.pop("index", None)
 		self.gestures = data.pop("gestures", {})
+		if self.text:
+			self.text = self.text.strip()
```

Comment:
@GatienBouyssou
Looks like a regression to me.
Seems like searching for "text " (excluding "text:" present earlier) is a pretty coherent criterion, yet rendered much more cumbersome to implement by this change.
What is the concrete case you're addressing here please?